content/java/concepts/errors/errors.md

```diff
@@ -14,19 +14,20 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-In Java, situations where things might go wrong in the program are predominantly handled by the following subclasses of the `java.lang.Throwable` class: `Error` and `Exception`.
+In Java, situations where [things might go wrong in the program](https://www.codecademy.com/resources/docs/general/error) are predominantly handled by the following subclasses of the `java.lang.Throwable` class: `Error` and `Exception`.
 
-The `Error` class represents illegal operations that cause serious abnormalities in the program and are not recommended to `catch`. Some examples include:
+The `Error` class represents illegal operations that cause serious abnormalities in the program and are not recommended to `catch`. Some examples include the following:
 
-- The `ClassFormatError`, which occurs when a class file cannot be read or interpreted.
-- The `IncompatibleClassChangeError`, which occurs when a base class is altered after a child class has already been initialized.
-- The `NoClassDefFoundError`, which occurs when the file with the class containing the `main()` method cannot be found.
+- The [`ClassFormatError`](https://www.codecademy.com/resources/docs/general/error/classformaterror) occurs when a class file cannot be read or interpreted.
```

Comment:
Given that this PR will merge new term entries for the "Java: Errors" (and thus create a new terms table on the bottom by default) do we really need a list here that might be repeating information?

Reply:
I think it might be good, just to make the distinction between errors and exceptions, that the term list isn't going to do for us. And eventually there will be a lot more terms than we have in these lists.

Reply:
Yeah that is an unfortunate side-effect of the terms tables. Let's go with what we have right now and see how things grow from here? (Maybe engineering can address that issue with term table organization in a future version).